Subtester/Assets/Plugins/Android/gradleTemplate.properties

```diff
@@ -0,0 +1,9 @@
+org.gradle.jvmargs=-Xmx**JVM_HEAP_SIZE**M
```

Comment:
This is now needed for the resolver to work correctly - we need to have a template for `gradle.properties`, just like we have one for `mainTemplate.gradle` -> `build.gradle`.

Reply:
It also means we no longer have to set up the `android.useAndroidX=true` property manually every time, though